knative-operator/Gopkg.lock

```diff
@@ -56,14 +56,6 @@
   revision = "37c8de3658fcb183f997c4e13e8337516ab753e6"
   version = "v1.0.1"
 
-[[projects]]
```

Comment:
How is HEAD updated ? I can't see any change in Gopkg.toml reflecting the update.

Reply:
The commit message was likely bad. I just ran `dep ensure -v` on HEAD since that already had diffs.